keylime/cloud_verifier_tornado.py

```diff
@@ -856,6 +857,53 @@ async def invoke_provide_v(agent):
         asyncio.ensure_future(process_agent(agent, states.GET_QUOTE))
 
 
+async def invoke_notify_error(agent, tosend):
+    if agent is None:
+        raise Exception("Agent deleted while being processed")
```

Comment:
Which part should handle this exception?

Reply:
Yeah, just converted it to a warning.